lib/containers/k8s.pm

```diff
@@ -21,19 +21,28 @@ use registration qw(add_suseconnect_product get_addon_fullname);
 our @EXPORT = qw(install_k3s uninstall_k3s install_kubectl install_helm install_oc);
 
 sub install_k3s {
-    my $k3s_version = get_var("CONTAINERS_K3S_VERSION", "v1.23.6+k3s1");
-    record_info('k3s', $k3s_version);
-    assert_script_run("curl -sfL https://get.k3s.io | INSTALL_K3S_VERSION=$k3s_version sh -");
-    enter_cmd("k3s server &");
```

Comment:
Note: The install script starts a k3s-server by default, unless `INSTALL_K3S_SKIP_START` is set to true. See https://rancher.com/docs/k3s/latest/en/installation/install-options/#options-for-installation-with-script for more details.

The same comment is also present in line 34 now.